exercises/diamond/src/test/java/DiamondPrinterTest.java

```diff
@@ -9,17 +10,24 @@
 import static org.junit.Assert.assertThat;
 
 public final class DiamondPrinterTest {
-
+    private DiamondPrinter diamondPrinter;
+    
+    @Before
+    
```

Comment:
This blank line between annotation and method signature is atypical; please remove it. Thanks! ❤️